udev/systemd-udev-241.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "hwdata",
```

Comment:
hwdata not needed for systemd.

Reply:
So it is not part of the systemd-hwdb stuff? It looked like something udev uses. Sure, will remove if it is totally unrelated and unneeded.

Reply:
usbutils and pciutils uses it. Systemd has it's own [copy](/systemd/systemd/tree/master/hwdb) of `pci.ids` and `usb.ids.

Reply:
Okay. I removed it.